__tests__/github-mention-test.js

```diff
@@ -19,20 +20,60 @@ describe('Github Mention', function() {
   // If you are working on the algorithm itself, it is useful to be able to run
   // the complete flow that downloads the diff and subsequent blames. Since
   // doing http requests is unreliable in tests, it is disabled by default.
-  xit('CompleteFlow', function() {
-    mentionBot.enableCachingForDebugging = true;
-    var prs = [3238];
-    prs.forEach(function(i) {
-      console.log(
-        i,
-        mentionBot.guessOwnersForPullRequest(
-          'https://github.com/facebook/react-native',
+  xdescribe('CompleteFlow', function() {
+    it('Gets correct users with no config options', function() {
+      mentionBot.enableCachingForDebugging = true;
+      var prs = [3238];
+      prs.forEach(function(i) {
+        console.log(
           i,
-          'mention-bot',
-          'master',
-          {} //config
-        )
+          mentionBot.guessOwnersForPullRequest(
+            'https://github.com/facebook/react-native',
+            i,
+            'mention-bot',
+            'master',
+            {
+              userBlacklist: []
+            } //config
+          )
+        );
+      });
+    });
+
+    it('Messages 5 users from config option maxUsersToPing', function() {
+      mentionBot.enableCachingForDebugging = true;
+      var owners = mentionBot.guessOwnersForPullRequest(
+        'https://github.com/facebook/react-native',
+        3238,
+        'mention-bot',
+        'master',
+        {
+          maxReviewers: 5,
+          userBlacklist: []
+        } //config
+      );
+      expect(owners.length).toEqual(5);
+    });
+
+    it('Should contain testname in owners from whitelist', function() {
+      mentionBot.enableCachingForDebugging = true;
+      var owners = mentionBot.guessOwnersForPullRequest(
+        'https://github.com/facebook/react-native',
+        3238,
+        'mention-bot',
+        'master',
+        {
+          userBlacklist: [],
+          userWhitelist: [
```

Comment:
I would call it something else. It reads as if it's going to be just an array of strings like the previous one is. But, it's actually paths. After thinking about it exactly 5 seconds, I came up with `alwaysNotifyForPaths`, but I'm sure there's better wording.